libraries/AC_AttitudeControl/AC_PosControl_Sub.cpp

```diff
@@ -42,7 +42,7 @@ void AC_PosControl_Sub::set_alt_target_from_climb_rate(float climb_rate_cms, flo
 ///     actual position target will be moved no faster than the speed_down and speed_up
 ///     target will also be stopped if the motors hit their limits or leash length is exceeded
 ///     set force_descend to true during landing to allow target to move low enough to slow the motors
-void AC_PosControl_Sub::set_alt_target_from_climb_rate_ff(float climb_rate_cms, float dt, bool force_descend)
+void AC_PosControl_Sub::set_alt_target_from_climb_rate_ff(float climb_rate_cms, float dt, bool force_descend, bool high_jerk_z)
```

Comment:
This change shouldn't be necessary for copter/plane/etc

Reply:
The build fails with `./waf copter` if this is not there:
Waf: Leaving directory `/Users/aubrey/Documents/planck/code/ardupilot/build/Pixhawk1'
Build failed
 -> task in 'objs/AC_AttitudeControl' failed (exit status 1):
	{task 4516495680: cxx AC_PosControl_Sub.cpp -> AC_PosControl_Sub.cpp.0.o}